packages/connect/package.json

```diff
@@ -90,6 +91,7 @@
         "@trezor/trezor-user-env-link": "workspace:*",
         "@types/karma": "^6.3.8",
         "@types/parse-uri": "^1.0.2",
+        "babel-loader": "^9.1.3",
```

Comment:
this was previously taken from global?

Reply:
Yes, but I need it defined here because of use `yarn workspaces focus` also it's somehow recommend by yarn to have minimal amount of global dependecies in monorepo